sos/plugins/openvswitch.py

```diff
@@ -29,6 +31,36 @@ def setup(self):
             '/usr/local/var/log/openvswitch/',
         ]
 
+        dpdk_enabled = self.call_ext_prog(
```

Comment:
`legacy-3.9` does not include `call_ext_prog()` in the API.

If you need to run something during `setup()`, get the output returned but _not_ write that output to the archive, use `Plugin.exec_cmd()`. If you do need to save the output as well, use `Plugin.collect_cmd_output()`.

Reply:
I tested this with sos-3.6 which didn't have collect_cmd_output(), I'll change this.